sql/src/main/java/io/crate/analyze/SetStatementAnalyzer.java

```diff
@@ -42,23 +42,27 @@ private SetStatementAnalyzer() {
     }
 
     public static SetAnalyzedStatement analyze(SetStatement node, ParameterContext parameterContext) {
-        if (SetStatement.Scope.SESSION.equals(node.scope())) {
-            logger.debug("SET SESSION STATEMENT: {}", node.toString());
+        if (!SetStatement.Scope.GLOBAL.equals(node.scope())) {
+            logger.warn("SET STATEMENT WITH SESSION OR LOCAL WILL BE IGNORED: {}", node);
```

Comment:
Wouldn't it be possible to return early here and just use an empty assignmentList? 
I'm not sure if it makes sense to have the verification further below that global settings don't conflict with session settings